lib/modules/manager/helmfile/schema.ts

```diff
@@ -1,25 +1,30 @@
 import { z } from 'zod';
+import { Yaml } from '../../../util/schema-utils';
 
-export const RepositorySchema = z.object({
+export const HelmRepository = z.object({
   name: z.string(),
   url: z.string(),
   oci: z.boolean().optional(),
 });
+export type HelmRepository = z.infer<typeof HelmRepository>;
 
-export const ReleaseSchema = z.object({
+export const HelmRelease = z.object({
   name: z.string(),
   chart: z.string(),
   version: z.string(),
   strategicMergePatches: z.unknown().optional(),
   jsonPatches: z.unknown().optional(),
   transformers: z.unknown().optional(),
 });
+export type HelmRelease = z.infer<typeof HelmRelease>;
 
-export const DocSchema = z.object({
-  releases: z.array(ReleaseSchema).optional(),
-  repositories: z.array(RepositorySchema).optional(),
+export const Doc = z.object({
```

Comment:
I'm not using `Yaml.pipe` here because `Doc` schema can further be used in the `extract.ts` file which doesn't use schema at the moment.